File: backend/src/main/java/top/vuhe/mapper/StatusMapper.java
package top.vuhe.mapper;

import com.baomidou.mybatisplus.core.mapper.BaseMapper;
import org.apache.ibatis.annotations.Mapper;
import org.springframework.stereotype.Repository;
import top.vuhe.entity.equipment.dao.StatusDAO;

/**
 * @author zhuhe
 */
@Mapper
@Repository
public interface StatusMapper extends BaseMapper<StatusDAO> {
}

File: backend-kt/src/main/kotlin/top/vuhe/sw/mapper/StatusMapper.kt
package top.vuhe.sw.mapper

import com.baomidou.mybatisplus.core.mapper.BaseMapper
import org.apache.ibatis.annotations.Mapper
import org.springframework.stereotype.Repository
import top.vuhe.sw.entity.equipment.dao.StatusDAO

@Mapper
@Repository
interface StatusMapper: BaseMapper<StatusDAO>